### DISCUSSION TITLE
Fail to re-verify academic affiliation

### DISCUSSION THREAD

**Discussion by gaomy3832** [community/community#51810] [GitHub Education]
I have been trying multiple times to re-verify my academic affiliation. I am a faculty at Tsinghua University in Beijing, China. I have fixed all the required fixes, including translating the proof to English, uploading a formal letter in addition to the ID card, and using the laptop build-in camera. Now it simply says the following general info without telling me why.

> Unfortunately, we weren't able to approve your educational discount request for ...
> What happened?
> Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.

I simply cannot know what is the problem and there is nothing I can do. This is really frustrating.

Please help with this case.

Attaching a screenshot of the rejection FYI. How do I suppose to proceed with this?!

![Screen Shot 2023-04-03 at 2 51 13 PM](https://user-images.githubusercontent.com/3386287/229433226-450eebf4-adbd-42b3-9a1f-540e2e9c1d61.png)

**Comment by Ntshangase** [comment]
Hey,

Make sure that when you are uploading documents the camera is of good quality, maybe logging in to GitHub with your phone can help with taking better quality pictures.

**Reply by gaomy3832** [reply]
Thanks for your kind suggestion. Yes I've already tried to use the phone camera but still not working. There were a few times where it was rejected because of the picture quality as said in the decision message, but most of the time the quality should be fine as far as I can tell.

**Reply by vivek7424** [reply]
![gitreverseshell php](https://user-images.githubusercontent.com/57445904/229506228-893ee730-3e4f-4115-9c05-476a81b0ddd6.GIF)

**Reply by vivek7424** [reply]
![gitreverseshell php](https://user-images.githubusercontent.com/57445904/229506228-893ee730-3e4f-4115-9c05-476a81b0ddd6.GIF)

**Reply by vivek7424** [reply]
![gitreverseshell php](https://user-images.githubusercontent.com/57445904/229506228-893ee730-3e4f-4115-9c05-476a81b0ddd6.GIF)

**Comment by Ntshangase** [comment]
Try signing out of GitHub then sign in again.

**Reply by gaomy3832** [reply]
Just tried, but still no luck ...

**Comment by TANG617** [comment]
Hi, have you solved the problem? I am also a student who has been troubled by the rejection ( for 17 times! )  and  I hope to know how you applied for the Developer Pack.

**Comment by samus-aran** [comment]
Hi @gaomy3832 ,

This post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

We can continue [your conversations in this discussion](https://github.com/orgs/community/discussions/58126). Thank you for helping us maintain a productive and tidy community for all our members.
